user_sync/rules.py

```diff
@@ -447,7 +464,14 @@ def add_dashboard_user(self, user_key, dashboard_connectors):
             owning_organization_info = self.get_organization_info(OWNING_ORGANIZATION_NAME)        
             desired_groups = owning_organization_info.get_desired_groups(user_key)
             groups_to_add = self.calculate_groups_to_add(owning_organization_info, user_key, desired_groups)
-            commands.add_groups(groups_to_add)
+
```

Comment:
Should have a helper function to add groups.    The function would group by type internally.